UnityProject/Assets/Scripts/Objects/Furniture/BuckleInteract.cs

```diff
@@ -17,6 +18,12 @@ public class BuckleInteract : MonoBehaviour, ICheckedInteractable<MouseDrop>, IC
 		private SpriteHandler occupiedSpriteHandler;
 		private Integrity integrity;
 
+		///<summary>
+		/// Events created to change the sprite of the roller bed
```

Comment:
Events are a bit universal in their use cases, so a better description would be:
"Event that get invoked when a player Buckles into an object that has this component"
and create two summaries for both events so the IDE picks them up.
"Event that get invoked when a player Unbuckles into an object that has this component"